### PR TITLE
Breadcrumbs

### PR DESCRIPTION
**Pull request by srkirkland**
this took way to long but turns out we were using the old style router, which also didn't have a good way to pass along custom route info that would be handy for breadcrumbs.  So, I rewrote all the routes, and then let each route specify the name it wanted as its title.  From there the new breadcrumb layout was simple.  I like the way it turned out.  

https://reactrouter.com/en/main/hooks/use-matches for info about the suggested way to do crumbs, which i mostly loosely followed

### CODE REVIEW

File: Finjector.Web/ClientApp/src/shared/Breadcrumbs.tsx [hunk 1]
```diff
@@ -1,34 +1,48 @@
...
+  const matches = useMatches();
 
-  if (paths.length === 0 || paths[0] === "locator") {
-    return null; // Don't render anything if we are on the homepage or the base /locator path
-  }
+  const matchesWithHandle = matches.filter((m) => m.handle);
+
+  const handleMatch = matchesWithHandle[matchesWithHandle.length - 1];
+
+  const hideBreadcrumbs = (handleMatch?.handle as { hideBreadcrumbs?: boolean })
```
Comment (laholstege) [inline Finjector.Web/ClientApp/src/shared/Breadcrumbs.tsx:13]: neat

**Review by laholstege — APPROVED** [review]
works well! couldnt find any issues while testing

**Comment by laholstege** [comment]
except you saved Breadcrumbs.tsx in /src/shared instead of /src/components/Shared

**Review by jSylvestre — CHANGES_REQUESTED** [review]
Create folder appears to be broken

**Review by jSylvestre — APPROVED** [review]

**Pull request merged**